pyproject.toml

```diff
@@ -11,18 +11,14 @@ include = ["cognite_toolkit/*", "cognite_toolkit/**/*"]
 [tool.poetry.dependencies]
 python = "^3.9"
 python-dotenv = "^1.0.0"
-cognite-sdk = {version = ">=7.20, <7.21", extras = ["pandas"]}
-pandas = "^2.1.1"
+cognite-sdk = {version = ">=7.24, <7.25", extras = ["pandas"]}
+pandas = ">=1.5.3, <3.0"
 pyyaml = "^6.0.1"
-chardet = "^5.1.0"
 typer = {version = "^0.9.0", extras = ["all"]}
 sentry-sdk = "^1.39"
-pyarrow = "^15.0.0"
-mypy = "^1.8.0"
-cognite-logger = "^0.6.0"
```

Comment:
These are dev dependencies and should not have been added here.

Reply:
cognite-logger is used in run function --local. It caught me by surprise, but if I remember correctly, it will fail the local environment creation without cognite-logger due to a dependency in the SDK.